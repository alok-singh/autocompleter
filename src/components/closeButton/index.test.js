import React from 'react';
import { render } from '@testing-library/react';
import CloseButton from './index';

test('Unit tests for close button', () => {
  expect(render(<CloseButton />).baseElement).toMatchSnapshot();
});
