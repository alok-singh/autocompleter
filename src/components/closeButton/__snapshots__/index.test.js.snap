// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit tests for close button 1`] = `
<body>
  <div>
    <div
      class="close-button"
    >
      <svg
        version="1.1"
        viewBox="0 0 357 357"
        x="0px"
        y="0px"
      >
        <g
          id="close"
        >
          <polygon
            points="357,35.7 321.3,0 178.5,142.8 35.7,0 0,35.7 142.8,178.5 0,321.3 35.7,357 178.5,214.2 321.3,357 357,321.3  214.2,178.5"
          />
        </g>
      </svg>
    </div>
  </div>
</body>
`;
